libs/vscode/tasks/src/lib/cli-task-commands.ts

```diff
@@ -105,7 +103,19 @@ function selectCliCommandAndShowUi(
   );
 }
 
-async function selectCliCommandAndPromptForFlags(command: string, projectName?: string) {
+async function selectCliCommandAndPromptForFlags(
+  command: string,
+  projectName?: string,
+  target?: string,
+  configuration?: string
+) {
+  let flags: string[] | undefined;
+  if (configuration) {
```

Comment:
Ok, I actually finally settled on `--configuration=production` because that will resolve the same as the other `run` command tasks to give us that "Task is already active" if something is already running with the same command and args.